server/routes/api/attachments.ts

```diff
@@ -71,15 +67,11 @@ router.post("attachments.create", auth(), async (ctx) => {
       maxUploadSize: process.env.AWS_S3_UPLOAD_MAX_SIZE,
       uploadUrl: endpoint,
       form: {
-        "Cache-Control": "max-age=31557600",
-        "Content-Type": contentType,
-        acl,
-        key,
-        policy,
-        "x-amz-algorithm": "AWS4-HMAC-SHA256",
-        "x-amz-credential": credential,
-        "x-amz-date": longDate,
-        "x-amz-signature": getSignature(policy),
+        ...{
+          "Cache-Control": "max-age=31557600",
+          "Content-Type": contentType,
+        },
```

Comment:
```suggestion
        "Cache-Control": "max-age=31557600",
        "Content-Type": contentType,
```

No need for the object spread here, right?